frog/imports/ui/App/FROGRouter.js

```diff
@@ -24,7 +24,9 @@ const Page = ({ isNotLoggedIn, isRedirect, isStudent, path, ready }) => {
     return <NotLoggedIn />;
   }
   if (!ready) {
-    return <p>Loading...</p>;
+    return process.env.NODE_ENV === 'production'
+      ? <img src="/images/Loading.gif" alt="" />
```

Comment:
You use everywhere `"images/Loading.gif"`
But the file you added is called `"Spinner.gif"`

Furthermore Clients will not be able to access a file if it is in `frog/client`. I believe you should put this file in `frog/public`